ladybug/psychrometrics.py

```diff
@@ -18,9 +18,19 @@ def saturated_vapor_pressure(t_kelvin):
 
     This function accounts for the different behaviour above vs. below
     the freezing point of water.
+
```

Comment:
What is the "expanded description"?  How do I add to it?

I included references because I really liked how the [PVLib repo](https://github.com/pvlib/pvlib-python) was making their docstrings.  All of their models were very understandable from the code thanks in large part to their descriptions and the fact that I could make sure the reference was the same as the model I was studying.

Reply:
See this page and look for `Note:`
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

It should be before Args.

Reply:
Thank you.  I switched all of the references to be under `Note:`